scripts/modify_lines.py

```diff
@@ -64,5 +71,8 @@ def limit_line_capacities(n, boi, max_limit):
 
     limit_line_capacities(n, boi, max_limit)
 
+    if snakemake.config.get('new_line', False):
```

Comment:
if we apply my suggestion where the new_line can be configured, this must change to

```suggestion
    if snakemake.config["lines"]["modify_lines"].get('new_line', False):
```